spec/fixtures/rails_users_app/Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM appmap-ruby_with_appmap:2.6
+ARG GEM_VERSION
+ARG RUBY_VERSION
 
-RUN apt-get update && apt-get install -y postgresql-client
+FROM appmap:${GEM_VERSION} as appmap
+
+FROM ruby:${RUBY_VERSION}
```

Comment:
Cool, I just learned about multi-stage builds.

Reply:
Yeah, they can be a big help.